setup.cfg

```diff
@@ -66,6 +95,9 @@ max_doc_length = 79
 # paths.
 ignore=CVS,.git,flake8.egg-info
 
+# Good variable names which should always be accepted, separated by a comma.
+good-names=i,j,k,ex,Run,_,pk,x,y,id, db
```

Comment:
This list seems a bit long, could it be reduced to `_,pk,x,y,id,db` (possibly with `ex,Run` if those are special too)?